bundle/Resources/config/services/command.yaml

```diff
@@ -0,0 +1,56 @@
+services:
```

Comment:
`commands.yaml` ? To be in line with other file names like event listeners, services, parameters, matchers.

Same goes for `controller.yaml` below.

Reply:
OK, I'll update (I followed the namespace structure BTW)